File: server/serverForGame/src/main/java/com/srvr/serverForGame/model/HouseBuildResponseDTO.java
package com.srvr.serverForGame.model;


import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class HouseBuildResponseDTO {
    private int stoimost_kvartyry;

    private int kolichestvo_kvartir;

    private int construction_time;

    private int ezemes_platez;

    private int player_capital;

}

File: AndroidClient/app/src/main/java/com/java/myapplication/data/HouseBuildResponseDTO.kt
package com.java.myapplication.data

import kotlinx.serialization.Serializable


@Serializable
data class HouseBuildResponseDTO(
    var stoimost_kvartyry: Int,
    var kolichestvo_kvartir: Int,
    var construction_time: Int,
    var ezemes_platez: Int,
    var player_capital: Int,
) {

}